src/templates/community-champion-profile.js

```diff
@@ -142,121 +143,75 @@ const ImageContainer = styled('div')`
+        font-size: ${NAME_FONT_SIZE};
+        line-height: ${NAME_LINE_HEIGHT};
+        margin-bottom: ${size.xsmall};
         text-align: center;
     }
-    @media ${screenSize.upToMedium} {
-        grid-template-columns: ${ICON_MOBILE_WIDTH} auto;
-    }
-`;
-
-const Icon = styled('img')`
-    padding-top: ${size.tiny};
-    width: 100%;
 `;
 
-const IconWithText = ({ alt, gridArea, src, text }) => (
-    <IconWithTextContainer gridArea={gridArea}>
-        <Icon src={src} alt={alt} />
-        <P2 collapse>{text}</P2>
-    </IconWithTextContainer>
-);
-
-const Occupation = ({ occupation }) => (
-    <IconWithText
-        gridArea="occupation"
-        src={BriefcaseIcon}
-        alt="Briefcase"
-        text={occupation}
-    />
-);
-
-const Location = ({ location }) => (
-    <IconWithText
-        gridArea="location"
-        src={LocationPinIcon}
-        alt="Location pin"
-        text={location}
-    />
-);
-
-const LanguagesTitle = styled(H6)`
-    color: ${({ theme }) => theme.colorMap.greyLightTwo};
-    font-size: ${fontSize.tiny};
-    font-weight: 500;
-    letter-spacing: 0.1em;
-    line-height: ${lineHeight.tiny};
-    text-transform: uppercase;
+const Bio = styled(P)`
+    margin-bottom: ${size.xlarge};
     @media ${screenSize.upToSmallDesktop} {
-        display: none;
+        margin-bottom: ${BIO_MOBILE_BOTTOM_MARGIN};
     }
 `;
 
-const LanguagesContainer = styled('div')`
-    grid-area: languages;
-`;
-
-const Languages = ({ languages }) => (
-    <LanguagesContainer>
-        <LanguagesTitle collapse>Languages</LanguagesTitle>
-        <IconWithText
-            src={SpeechBubbleIcon}
-            alt="Speech bubble"
-            text={languages}
-        />
-    </LanguagesContainer>
-);
-
-const InfoSection = ({ location, occupation, languages }) => (
-    <InfoGrid>
-        <Occupation occupation={occupation} />
-        <Location location={location} />
-        <Languages languages={languages} />
-    </InfoGrid>
-);
-
-const Name = styled(H1)`
+const Title = styled(H4)`
     margin-bottom: ${size.mediumLarge};
     @media ${screenSize.upToSmallDesktop} {
-        font-size: ${NAME_FONT_SIZE};
-        line-height: ${NAME_LINE_HEIGHT};
-        margin-bottom: ${size.xsmall};
-        text-align: center;
+        margin-bottom: ${size.default};
+    }
+    @media ${screenSize.upToMedium} {
+        font-size: ${fontSize.medium};
+        line-height: 30px;
     }
 `;
 
-const Bio = styled(P)`
-    margin-bottom: ${size.xlarge};
+const CertificationsContainer = styled('div')`
+    column-gap: ${CERTIFICATIONS_CONTAINER_COLUMN_GAP};
+    display: flex;
+    flex-wrap: wrap;
+    row-gap: ${size.default};
     @media ${screenSize.upToSmallDesktop} {
-        margin-bottom: ${BIO_MOBILE_BOTTOM_MARGIN};
+        column-gap: ${size.large};
+        margin-bottom: ${size.xlarge};
     }
 `;
 
+const CertificationsSection = ({ certifications }) => {
+    const badges = [
```

Comment:
Let's try to handle this however you see fit (where these could be `null` or `undefined`